src/www/ui/api/Controllers/UploadController.php

```diff
@@ -837,4 +837,20 @@ public function getGroupsWithPermissions($request, $response, $args)
     $res["permGroups"] = $finalPermGroups;
     return $response->withJson($res, 200);
   }
+
+  public function getReuseReportSummary($request, $response, $args)
+  {
+    $uploadId = intval($args['id']);
+
+    if (!$this->dbHelper->doesIdExist("upload", "upload_pk", $uploadId)) {
+      $returnVal = new Info(404, "Upload does not exist", InfoType::ERROR);
+      return $response->withJson($returnVal->getArray(), $returnVal->getCode());
+    }
+
+    $symfonyRequest = new \Symfony\Component\HttpFoundation\Request();
+    $symfonyRequest->request->set('upload', $uploadId);
+    $ajaxReuseReport = $this->restHelper->getPlugin('ajax_reuse_report');
```

Comment:
Can use direct method from library: `\Fossology\Lib\BusinessRules\ReuseReportProcessor::getReuseSummary()`

It provides the static (const) keys as well which are more helpful in REST API than UI titles.

Reply:
Done